### DISCUSSION TITLE
What would you change in the app design?

### DISCUSSION THREAD

**Discussion by gokadzev** [gokadzev/Musify#324] [Show and tell]
Share your ideas!

**Comment by YisusChrist** [comment]
Hello,

I don't know if it only occurs to me but the bottom buttons of the play interface are very small, making it a little bit difficult to press them properly.

![Screenshot_2023-12-16-11-16-01-39_60d40bf597786c9bfd90ecfc534f1982~2.jpg](https://github.com/gokadzev/Musify/assets/58041337/eef4e56b-9d2b-4a13-aa9b-ab9b42b33790)

I think that it would be better to make them bigger and reorganize the 4 external buttons (2 on the left and 2 on the right) in any other position.

Another thing that could be improved is how to access the play interface. Currently, when you tap the bottom bar that displays the current song information nothing happens and you are forced to press the ⬆️ button on the left to access the interface. I'm sure it is way more intuitive and easy to just make all the bar clickable and remove the ⬆️ button.

![Screenshot_2023-12-16-11-23-32-35_60d40bf597786c9bfd90ecfc534f1982.jpg](https://github.com/gokadzev/Musify/assets/58041337/65de4225-754b-4de3-82f9-2ae13adab033)

**Reply by gokadzev** [reply]
> I don't know if it only occurs to me but the bottom buttons of the play interface are very small, making it a little bit difficult to press them properly.

I will consider that for the next release.

> Another thing that could be improved is how to access the play interface. Currently, when you tap the bottom bar that displays the current song information nothing happens and you are forced to press the ⬆️ button on the left to access the interface. I'm sure it is way more intuitive and easy to just make all the bar clickable and remove the ⬆️ button.


Possible, but some users will be confused as there will be no sign that it's clickable and can be expanded.

**Comment by OmarNazzal150** [comment]
Hi. On the home screen of musify. The suggestion artists don't have a 3 dots button to view more suggested artists, just like the suggestion Playlist has to press the 3 dot button to view more suggested playlists



![Screenshot_٢٠٢٣١٢١٦_١٦١١١٢.jpg](https://github.com/gokadzev/Musify/assets/135146653/043cc264-0fb2-4d08-9c9e-4fc0ea519b2d)

**Reply by gokadzev** [reply]
> Hi. On the home screen of musify. The suggestion artists don't have a 3 dots button to view more suggested artists, just like the suggestion Playlist has to press the 3 dot button to view more suggested playlists
> 
> ![Screenshot_٢٠٢٣١٢١٦_١٦١١١٢.jpg](https://private-user-images.githubusercontent.com/135146653/290997611-043cc264-0fb2-4d08-9c9e-4fc0ea519b2d.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDI3NTEyODksIm5iZiI6MTcwMjc1MDk4OSwicGF0aCI6Ii8xMzUxNDY2NTMvMjkwOTk3NjExLTA0M2NjMjY0LTBmYjItNGQwOC05YzllLTRmYzBlYTUxOWIyZC5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMjE2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTIxNlQxODIzMDlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zMzE1YzJkMDYzOGMwNzY3MTRlNzhlNGIxZjBlNjllZTUyMzcxMWYzNjI5NWQwMDM5MzI3M2RlOWFjOTk3MWZjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.qdFtO9PEbyBh7J7F3okLLNVG4eZJCoq8OpwXsdhCp4w)

Basically they're extracted from only top music list, but yeah it may be added.

**Reply by OmarNazzal150** [reply]
Thanks
